worlds/hk/Items.py

```diff
@@ -19,18 +19,38 @@ class HKItemData(NamedTuple):
 for item, item_data in item_table.items():
     lookup_type_to_names.setdefault(item_data.type, set()).add(item)
 
-item_name_groups = {group: lookup_type_to_names[group] for group in ("Skill", "Charm", "Mask", "Vessel",
-                                                                     "Relic", "Root", "Map", "Stag", "Cocoon",
-                                                                     "Soul", "DreamWarrior", "DreamBoss")}
-
 directionals = ('', 'Left_', 'Right_')
-
-item_name_groups.update({
+item_name_groups = ({
+    "BossEssence": lookup_type_to_names["DreamWarrior"] | lookup_type_to_names["DreamBoss"],
+    "BossGeo": lookup_type_to_names["Boss_Geo"],
+    "CDash": {x + "Crystal_Heart" for x in directionals},
+    "Charms": lookup_type_to_names["Charm"],
+    "CharmNotches": lookup_type_to_names["Notch"],
+    "Claw": {x + "Mantis_Claw" for x in directionals},
+    "Cloak": {x + "Mothwing_Cloak" for x in directionals} | {"Shade_Cloak", "Split_Shade_Cloak"},
+    "LifebloodCocoons": lookup_type_to_names["Cocoon"],
     "Dreamers": {"Herrah", "Monomon", "Lurien"},
-    "Cloak": {x + 'Mothwing_Cloak' for x in directionals} | {'Shade_Cloak', 'Split_Shade_Cloak'},
-    "Claw": {x + 'Mantis_Claw' for x in directionals},
-    "CDash": {x + 'Crystal_Heart' for x in directionals},
     "Fragments": {"Queen_Fragment", "King_Fragment", "Void_Heart"},
```

Comment:
Would probably be better for this to be WhiteFragments to distinguish from vessel fragments, while we're changing things